src/styles/blocks/catalog.scss

```
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform 0.3s linear, color 0.3s linear;
+
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(120%);
+      color: #34568b;
```

Create variables for colors
```suggestion
      color: $c-someColor;
```